docs/articles/documentation/templates/actions/basic-action-options.md

```diff
@@ -0,0 +1,30 @@
+# Basic Action Options
+
+Basic action options provide additional parameters for the `t.pressKey`, `t.selectText`, `t.selectTextAreaContent` and `t.selectEditableContent` actions.
```

Comment:
# Template Headers
In all option templates, the header can just be `# Options` because the templates will be inserted to the topic they apply to. And we don't need the opening phrase as well for the same reason.
And mind the header level - it should be `##` at least.

Reply:
@VasilyStrelyaev Some methods include several options templates to be included. Simply naming them `Options` may not be such a good idea in this case. 
Therefore, is it possible that we do not rename them?

Reply:
As discussed, `Selector Options` will be renamed to `Select Target Elements` and the specific action options will be `Options`.